src/js/interim.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * The following theme has transitional properties to
+ * mitigate the upgrade to the design-system theme.
+ * Using the interim theme will help you avoid critical
+ * and immediate breaking changes.
+ *
+ * How does it work?
+ * Copy the content of the interimTheme file to your local theme file
```

Comment:
Rather than "Copy", can we set it up so that:
`import { v2migration } from 'grommet-theme-hpe';`
works?

Reply:
...And then you are getting into a potential 3 layer theming (hpe+v5Migration+product themes), and you need to document how to use the deepMerge in between the layers. 
I thought about it too at the beginning and decided to keep the 'Copy' instead.
I don't think this theme will be used much, so I prefer to keep the usage of it simple.

Reply:
Why does the caller need to do `deepMerge`? I was trying to keep the caller's experience as simple as possible:
```
import { v2migration } from 'grommet-theme-hpe';
<Grommet theme={v2migration} />
```
which would allow us to do anything we needed to inside the `v2migration` theme.

Reply:
In your example, you are assuming there is no local theme for the specific product, which is not the case in most scenarios.

In addition, What about the hpe theme? 
```
import { v2migration, hpe } from 'grommet-theme-hpe';
```
we still want the user to import the hpe theme, so it will be easier to separate the themes and understand each theme's role while gradually eliminating styles from the `v2migration` theme.
Eliminating the styles gradually from the `v2migration` theme will be easier if you copy the styles to your local file and take them off one by one.

Reply:
I'm assuming any local theme is handled the same way it is handled today, perhaps merging with theme from grommet-theme-hpe. So, no change in that regard. And, I'm assuming that `v2migration` has the `hpe` theme included in it. So, you use `v2migration` until you are done and then switch to the latest `hpe`. But, I'm happy to defer to you on this, as you've been helping the Green Lake team out directly.